src/SFA.DAS.Apprenticeships.Web/Views/ChangeOfStartDate/Provider/CheckDetails.cshtml

```diff
@@ -1,13 +1,64 @@
 ﻿@using SFA.DAS.Apprenticeships.Web.Models.ChangeOfStartDate
 @model ProviderChangeOfStartDateModel
 @{
-	ViewData["Title"] = "Check your changes before sending to the employer - Manage apprenticeships";
-	var editPage = $"/provider/{Model.ProviderReferenceNumber}/ChangeOfPrice/{Model.ApprenticeshipHashedId}/edit?cacheKey={Model.CacheKey}";
+    ViewData["Title"] = "Check your changes before sending to the employer - Manage apprenticeships";
+    var editPage = $"/provider/{Model.ProviderReferenceNumber}/ChangeOfStartDate/{Model.ApprenticeshipHashedId}/edit?cacheKey={Model.CacheKey}";
 }
 
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <h1 class="govuk-heading-l">Check your change before sending to the employer</h1>
+        <h2 class="govuk-heading-m">Change you're requesting</h2>
+        <table class="govuk-table">
+            <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+                <th scope="col" class="govuk-table__header app-custom-class govuk-!-width-one-third"/>
+                <th scope="col" class="govuk-table__header app-custom-class">Current</th>
+                <th scope="col" class="govuk-table__header app-custom-class govuk-!-width-one-third">Requested</th>
+                <td class="govuk-table__cell"></td>
+            </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+            <tr class="govuk-table__row">
```

Comment:
We have components we can use here in place of the row e.ge
                    <component type="typeof(DetailsRowComponent)"
                               param-Id='"TrainingStartDate"'
                               param-Label='"Training start date"'
                               param-OriginalValue='Model.OriginalActualStartDate.ToString("dd MMMM yyyy")'
                               param-Value='Model.PendingActualStartDate.ToString("dd MMMM yyyy")'
                               render-mode="Static" />